charts/kyverno/templates/hooks/post-upgrade-migrate-resources.yaml

```diff
@@ -104,6 +104,23 @@ spec:
             - {{ . }}
             {{- end }}
           {{- with .Values.crds.migration.securityContext }}
+          env:
+          - name: KYVERNO_SERVICEACCOUNT_NAME
+            value: {{ template "kyverno.fullname" . }}-migrate-resources
+          - name: KYVERNO_DEPLOYMENT
+            value: {{ template "kyverno.admission-controller.name" . }}
```

Comment:
This is not the admission controller, what is used for?

Reply:
@realshuting  I don't know if this is needed. Added because it is the one from the list of validated env vars you provided.